doc/manual/packages.rst

```diff
@@ -621,11 +629,22 @@ not domain experts*.
 
   * ``MATLAB.jl`` provides an interface to call the MATLAB engine from within Julia.
 
+6. Creative names are cool, but confusion isn't so much.
+
+  * Many modern programming languages such as Ruby and Python have popular packages
+    that have entertaining names. Python's ``pickle`` package isn't about pickles, it's
+    for serializing (or pickling) object structures. Ruby's ``Sinatra`` package is named
+    after the popular American singer and is a popular web framework that competes with Ruby's
+    ``Rails`` web framework (another creative name).
+
+  * If being a creative package namer isn't your style, it's okay to not force it.
```

Comment:
Here's a result for searching on `pickle`
![screen shot 2015-06-16 at 11 34 33 am](https://cloud.githubusercontent.com/assets/324836/8187075/eecc4de0-141b-11e5-8c97-6a08cfc1636b.png)